mmdet/datasets/openimages.py

```diff
@@ -0,0 +1,845 @@
+            label_description_file (str): File path to the label map proto.
+
+        Returns:
+            list[str]: Class name of OpenImages.
+        """
+
+        index_list = []
+        classes_names = []
+        with open(label_description_file, 'r') as f:
+            reader = csv.reader(f)
+            for line in reader:
+                self.cat2label[line[0]] = line[1]
+                classes_names.append(line[1])
+                index_list.append(line[0])
+        self.index_dict = {index: i for i, index in enumerate(index_list)}
+        return classes_names
+
+    def list_from_csv(self, ann_file):
+        """Load annotation from csv style ann_file.
+
+        Args:
+            ann_file (str): CSV style annotation file path.
+
+        Returns:
+            tuple: Returns (item_list, data_infos), where
+
+            `item_list` (defaultdict[list[dict]]):
+            Annotations where item of the defaultdict indicates an
+            image, each of which has (n) dicts. Keys of dicts are:
+
+                - `bbox` (list): coordinates of the box, in normalized image
+                    coordinates, of shape 4.
+                - `label` (int): the label id.
+                - `is_group_of` (bool):  Indicates that the box spans a group
+                    of objects (e.g., a bed of flowers or a crowd of people).
+                - `is_occluded` (bool): Indicates that the object is occluded
+                    by another object in the image.
+                - `is_truncated` (bool): Indicates that the object extends
+                    beyond the boundary of the image.
+                - `is_depiction` (bool): Indicates that the object is a
+                    depiction.
+                - `is_inside` (bool): Indicates a picture taken from the
+                    inside of the object.
+            `data_infos` (list[dict])： Data infos where each item of
+            the list indicates an image. Keys of annotations are:
+
+                - `img_id` (str): Image name.
+                - `filename` (str): Image name with suffix.
+        """
+
+        item_list = defaultdict(list)
+        data_infos = []
+        cp_filename = None
+        with open(ann_file, 'r') as f:
+            reader = csv.reader(f)
+            i = -1
+            for line in reader:
+                i += 1
+                if i == 0:
+                    continue
+                else:
+                    img_id = line[0]
+                    filename = f'{img_id}.jpg'
+                    label_id = line[2]
+                    assert label_id in self.index_dict
+                    label = int(self.index_dict[label_id])
+                    bbox = [
+                        float(line[4]),  # xmin
+                        float(line[6]),  # ymin
+                        float(line[5]),  # xmax
+                        float(line[7])  # ymax
+                    ]
+                    is_occluded = True if int(line[8]) == 1 else False
```

Comment:
May add description about the file format in the code in this PR. And add documentation after this PR.

Reply:
After fixing, have already described in `load_annotations`